content/bn/client-server-architecture.md

```diff
@@ -0,0 +1,36 @@
+---
+title: ক্লায়েন্ট-সার্ভার স্থাপত্য (Client-Server Architecture)
+status: Completed
+category: ধারণা
+tags: ["স্থাপত্য", "", ""]
```

Comment:
Kindly don't translate tags. Keep then as they were in English

Reply:
Ok, I will keep that in mind. @mitul3737